### PR TITLE
Refactor test, implement dual banner fix

### PR DESCRIPTION
**Pull request by housepbass**
Implement handling for multiple banner configurations. ie a configuration file containing both `banner exec` and `banner motd`

### CODE REVIEW

File: tests/unit/mock/config/compliance/feature_compliance/ios_dual_banner_received.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 data = {
     "compliant": False,
-    "missing": "banner exec ^C\n=========\nintended config exec banner\n-========^C\n",
-    "extra": "banner exec ^C\n=========\nactual config exec banner\n-========^C\n",
+    "missing": "banner exec ^C\n=========\nintended config exec banner\n-========^C",
+    "extra": "banner exec ^C\n=========\nactual config exec banner\n-========^C",
     "cannot_parse": True,
     "unordered_compliant": False,
     "ordered_compliant": False,
-    "actual": "hostname dual-banner\n!\nbanner exec ^C\n=========\nactual config exec banner\n-========\n^C\nbanner motd ^C\n======\nactual config motd banner\n======\n   || ($hostname) ||\n^C\n!\n",
```
Comment (itdependsnetworks) [inline tests/unit/mock/config/compliance/feature_compliance/ios_dual_banner_received.py:8]: I think we need an additional test, if you only test for `banner` (and not `banner exec` does this capture both and work correctly?

Comment (housepbass) [reply tests/unit/mock/config/compliance/feature_compliance/ios_dual_banner_received.py:8]: Good point, I'll add one that checks `banner` and another that checks `banner motd`

Comment (housepbass) [reply tests/unit/mock/config/compliance/feature_compliance/ios_dual_banner_received.py:8]: @itdependsnetworks I added `banner`, `banner exec`, `banner motd` tests.

**Review by jeffkala — APPROVED** [review]
looks good

**Review by itdependsnetworks — APPROVED** [review]

**Review by qduk — APPROVED** [review]

**Pull request merged**